h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.template.html

```diff
@@ -50,6 +50,7 @@
   <clr-dg-row *ngFor="let item of vchs">
     <clr-dg-action-overflow class="{{item.name}}">
       <button class="action-item {{item.name}}" (click)="deleteVch(item)">Delete</button>
+      <button class="action-item cert {{item.name}}" (click)="downloadVchCert(item)">Download VCH certificate</button>
```

Comment:
This is not a major issue but potentially makes our tests brittle. Since now have more than one button element with the same class (which basically is the VCH name), you need to be aware that altering the order of these buttons will break the E2E tests. See the attached snippet below:

https://github.com/vmware/vic-ui/blob/master/h5c/vic/src/vic-webapp/e2e/app.po.ts#L147

I suggest you add a class to it just like you did for downloadVchCert.

Reply:
To give an extra context, the default behavior of `clickByCSS()`, as you wrote, will pick up whatever is found first if multiple instances are found.